waterlog/src/App.scss

```diff
@@ -2,216 +2,189 @@
 @import './Stylesheets/wastage';
 @import './Stylesheets/alerts';
 @import './Stylesheets/wastageSummary';
-@import './Stylesheets/tank';
 @import './Stylesheets/nav';
 @import './Stylesheets/tankInfo';
 @import './Stylesheets/progressBar';
 @import './Stylesheets/segmentLeak';
-@import './Stylesheets/histroryTable';
 @import './Stylesheets/map';
 @import './Stylesheets/sensor';
 @import './Stylesheets/error404';
 @import './Stylesheets/alertsFilter';
 
-.App-header {
-  position: fixed;
-  top: 0;
-  width: 100%;
-  z-index: 5;
+hr {
+	margin: 0 20px;
 }
 
-.router-outlet {
-  margin-top: 44px;
+button {
+	width: 240px;
+	height: 40px;
+	border-radius: 5px;
+	text-transform: uppercase;
+	padding: 2px 10px;
+	border: none;
 }
 
 #cost-buttons {
-  padding-top: 18px;
+	padding-top: 18px;
 }
 
 .btn-graph-nav {
-  height: 26px;
-  width: 84px;
+	height: 26px;
+	width: 84px;
 }
 
 .divide {
-  margin-top: 20px;
+	margin-top: 20px;
 }
 
 .bold {
-  font-weight: bold;
+	font-weight: bold;
 }
 
 .wastageInfo {
```

Comment:
inconsistent naming scheme `.btn-graph-nav` vs `.wastageInfo` your team needs to choose one and use it in all css files